scripts/release_ppa.sh

```diff
@@ -41,33 +47,35 @@ else
     branch=$1
 fi
 
-if [ "$branch" = develop ]
-then
-    pparepo=ethereum-dev
-    ppafilesurl=https://launchpad.net/~ethereum/+archive/ubuntu/ethereum-dev/+files
-else
-    pparepo=ethereum
-    ppafilesurl=https://launchpad.net/~ethereum/+archive/ubuntu/ethereum/+files
-fi
-
 keyid=70D110489D66E2F6
 email=builds@ethereum.org
 packagename=solc
 
-for distribution in trusty xenial bionic cosmic
+for distribution in xenial bionic cosmic static
 do
 cd /tmp/
 rm -rf $distribution
 mkdir $distribution
 cd $distribution
 
-# Dependency
-if [ $distribution = trusty -o $distribution = vivid ]
+if [ $distribution = static ]
 then
+    ppafilesurl=https://launchpad.net/~ethereum/+archive/ubuntu/ethereum-static/+files
+    pparepo=ethereum-static
     Z3DEPENDENCY=""
+    CMAKE_OPTIONS="-DSOLC_LINK_STATIC=On"
 else
+    if [ "$branch" = develop ]
+    then
+        pparepo=ethereum-dev
+        ppafilesurl=https://launchpad.net/~ethereum/+archive/ubuntu/ethereum-dev/+files
+    else
+        pparepo=ethereum
+        ppafilesurl=https://launchpad.net/~ethereum/+archive/ubuntu/ethereum/+files
```

Comment:
Can't we derive that from `$pparepo` for all three cases?

Reply:
Used for the ``dput`` call at the very end.

Reply:
We indeed can - done.